identity/handler.go

```diff
@@ -159,10 +159,7 @@ type listIdentitiesParameters struct {
 func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	page, itemsPerPage := x.ParsePagination(r)
 
-	params := ListIdentityParameters{Expand: ExpandDefault, Page: page, PerPage: itemsPerPage, CredentialsIdentifier: r.URL.Query().Get("credentials_identifier")}
-	if params.CredentialsIdentifier != "" {
-		params.Expand = ExpandEverything
-	}
+	params := ListIdentityParameters{Expand: ExpandEverything, Page: page, PerPage: itemsPerPage, CredentialsIdentifier: r.URL.Query().Get("credentials_identifier")}
```

Comment:
ExpandEverything is very slow because it fetches from several tables. It has to be optional